src/components/Pagination.js

```diff
@@ -0,0 +1,47 @@
+import pageBackIcon from "../assets/icon/ic_pagination_left_dark.svg";
+import pageNextIcon from "../assets/icon/ic_pagination_right_dark.svg";
+import "./Pagination.css";
+
+function Pagination({ totalPage, currentPage, setCurrentPage }) {
+  const maxPage = totalPage >= 5 ? 5 : totalPage;
+  const pages = Array.from({ length: maxPage }, (_, index) => index + 1);
+
+  const handlePageClick = (e) => {
+    const nextPage = Number(e.target.innerText);
+    setCurrentPage(nextPage);
+  };
```

Comment:
이곳도 위에서 설명드린것처럼 e.target.innerText 보다는 파라미터로 e 대신 page 숫자를 바로 받아올 수 있게 구현해보세요!

Reply:
프로퍼티 값 안의 객체도 계속 구조 분해 할 수 있다는 걸 처음 알았네요!

[refactor(Pagination-handling): Get specific property as parameter instead of event object](https://github.com/un0211/6-Sprint-Mission/commit/7025e34b675c492b0380b9189979376a2236233f)